backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/common/utils/ansible/AnsibleCommandConfig.java

```diff
@@ -307,14 +301,16 @@ private File setupAnsibleRunnerExecutionDir() throws IOException {
     }
 
     private void createHostFile(File inventory) {
-        File hostsFile = new File(String.format("%1$s/hosts", inventory));
-        if (CollectionUtils.isNotEmpty(hostnames())) {
+        File hostFile = new File(String.format("%1$s/hosts", inventory));
+        if (!hostname.isEmpty()) {
```

Comment:
Can it be null? `!StringUtils.isEmpty(hostname)` may be better.

Reply:
I removed this property completely now